main.js

```diff
@@ -34,11 +34,14 @@ module.exports = async ({ github, context, core }) => {
   console.log("pull request found, setting outputs");
 
   const name = author.name || author.login;
+  const twitter_username = author.twitter_username
+    ? author.twitter_username
+    : "";
```

Comment:
My knowledge of javascript is pretty limited, but would
```suggestion
  const twitter_username = author.twitter_username || ""
```
work?

Reply:
Yeah that work as well!